vita3k/gui/src/main_menubar.cpp

```diff
@@ -26,10 +26,16 @@
 namespace gui {
 
 static void draw_file_menu(GuiState &gui, EmuEnvState &emuenv) {
-    auto lang = gui.lang.main_menubar.file;
+    const auto TEXTURE_PATH{ emuenv.shared_path / "textures" };
+
+    auto &lang = gui.lang.main_menubar.file;
     if (ImGui::BeginMenu(lang["title"].c_str())) {
         if (ImGui::MenuItem(lang["open_pref_path"].c_str()))
             open_path(emuenv.pref_path.string());
+        if (ImGui::MenuItem(lang["open_texture_folder"].c_str())) {
+            if (fs::exists(TEXTURE_PATH))
```

Comment:
Why is the order reversed?

Reply:
You mean fs::exist needs to before the menu item?

but this line just to test, need create dir in app init of a new PR

Reply:
ah ok
i think reverse would be more correct anyway

Reply:
> You mean fs::exist needs to before the menu item? but this line just to test, need create dir in app init of a new PR this line will be delete

Yes, all time write check before menu 🫡, 1nd set both is same line with &&

Reply:
> Yes, all time write check before menu 🫡, and set both is same line with &&

but actually we want create dir in app init, no need check exist